cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/log/LogHandlerTestRun.java

```diff
@@ -456,17 +511,44 @@ private void verifyLogs(HttpResponse response, String entityId, String format, b
       } else {
         expectedStr += "&lt;img&gt;-" + expected;
       }
-      Assert.assertEquals(expectedStr, log.substring(log.length() - expectedStr.length()));
+      if (!Strings.isNullOrEmpty(log)) {
```

Comment:
I see, then it is okay. But look at my other comment https://github.com/caskdata/cdap/pull/6507#discussion_r75445698